docs/topics/spider-middleware.rst

```diff
@@ -24,20 +24,21 @@ Here's an example::
         'myproject.middlewares.CustomSpiderMiddleware': 543,
     }
 
-The specified :setting:`SPIDER_MIDDLEWARES` setting is merged with the default
-one (i.e. it does not overwrite it) and then sorted by order to get the final
-sorted list of enabled middlewares: the first middleware is the one closer to
-the engine and the last is the one closer to the spider.
-
-To decide which order to assign to your middleware see the default
-:setting:`SPIDER_MIDDLEWARES` setting and pick a value according to where
-you want to insert the middleware. The order does matter because each
+The :setting:`SPIDER_MIDDLEWARES` setting is merged with the
+:setting:`SPIDER_MIDDLEWARES_BASE` setting defined in Scrapy (and not meant to
+be overridden) and then sorted by order to get the final sorted list of enabled
+middlewares: the first middleware is the one closer to the engine and the last
+is the one closer to the spider.
+
+To decide which order to assign to your middleware see the
+:setting:`SPIDER_MIDDLEWARES_BASE` setting and pick a value according to where
 middleware performs a different action and your middleware could depend on some
```

Comment:
Line 34 from the original version got deleted (https://github.com/scrapy/scrapy/pull/1586/files#diff-17a393aa4b57b33782da9b32c3422e73L34). It should be "<...> and pick a value according to where you want to insert the middleware. The order does matter because each middleware performs <...>".